.github/workflows/check-unity-sdk-for-updates.yml

```
@@ -0,0 +1,55 @@
+          token: ${{ secrets.PUBLIC_REPO_BOT_GITHUB_TOKEN }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Check for AdGem SDK updates
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:52: Double quote to prevent globbing and word splitting

(shellcheck)